data/wof-schema.sql

```diff
@@ -11,3 +12,13 @@ CREATE TABLE wof_neighbourhood (
 CREATE INDEX wof_neighbourhood_wof_id_index ON wof_neighbourhood(wof_id);
 CREATE INDEX wof_neighbourhood_label_position_index ON wof_neighbourhood USING GIST(label_position);
 CREATE INDEX wof_neighbourhood_geometry_index ON wof_neighbourhood USING GIST(geometry);
+
+CREATE TABLE wof_neighbourhood_placetype (
+  placetype_code SMALLINT UNIQUE NOT NULL,
+  placetype_string text NOT NULL
+);
+
+INSERT INTO wof_neighbourhood_placetype VALUES
+  (1, 'neighbourhood'),
```

Comment:
I'd probably have it be driven off of which layer it gets shown in. If it's not in `places`, then I'd start creating other tables for it. Or consider a more general structure for it if that makes more sense.